tests/suites/test_suite_mpi.function

```diff
@@ -712,8 +712,8 @@ exit:
 /* END_CASE */
 
 /* BEGIN_CASE */
-void mbedtls_mpi_cmp_mpi( char * input_X, char * input_Y,
-                          int input_A )
+void mpi_cmp_mpi( char * input_X, char * input_Y,
```

Comment:
CI is failing due to `generate_bignum_tests.py` generating test cases using the old name of this test function (also `mpi_add_mpi` and `mpi_cmp_abs`). Removing `mbedtls_` from the `test_function` attributes in the script should resolve it.